readthedocs/urls.py

```diff
@@ -80,8 +75,14 @@
 
 # Export URLs
 groups = [basic_urls, rtd_urls, project_urls, api_urls, core_urls, i18n_urls,
-          money_urls, deprecated_urls]
-
+          deprecated_urls]
+
+if getattr(settings, 'INCLUDE_DONATIONS', True):
```

Comment:
I feel like this is an unneeded pattern that is replicating part of what ``INSTALLED_APPS`` should provide